nuxeo-drive-client/nxdrive/manager.py

```diff
@@ -6,6 +6,10 @@
 import sys
 import logging
 from urlparse import urlparse
+import urllib2
+import js2py.pyjs
```

Comment:
Need to import js2py.pyjs explicitly so that build script will include this module while packaging.